impl/src/main/java/io/jsonwebtoken/impl/DefaultJwtParser.java

```diff
@@ -257,6 +257,11 @@ public Jwt parse(String jwt) throws ExpiredJwtException, MalformedJwtException,
 
         Assert.hasText(jwt, "JWT String argument cannot be null or empty.");
 
+        if ("..".equals(jwt)) {
```

Comment:
Why was this added?  Seems to very special case that is handled by standard processing later.

Reply:
My understanding in the discussion above was that `..` is not a valid JWS, because it needs to have at least a header.
Without this check, `..` is accepted.